### ISSUE TITLE
Active NavLink in a Navbar doesn't have aria-current="page"

### ISSUE THREAD

**Issue by mubaraqwahab** [react-bootstrap/react-bootstrap#5423]
## Is your feature request related to a problem? Please describe

I noticed that an active `NavLink` in a `Navbar` doesn't render with an `aria-current="page"` attribute identifying it as a link to the current page. I don't know if this is a "problem" exactly though.

## Describe the solution you'd like

An active `NavLink` in a `Navbar` should have an `aria-current="page"` attribute.

Something like this (maybe) in [`AbstractNavItem`](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/AbstractNavItem.tsx#L92):

```js
props["aria-current"] = isActive && "page" 
```

## Describe alternatives you've considered

Bootstrap does something similar in their [navbar doc](https://getbootstrap.com/docs/4.5/components/navbar/):

```html
<a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
```

Reach Router's `Link` also adds the attribute automatically.

**Comment by jquense** [comment]
I'm not sure we can do this automatically. Isn't the `aria-current` context specific to the link being for the actual current page? We don't know if the link rendered is for the current page or not, or even if ultimately renders an anchor at all. I think this is more the job for a Router link component (as you noted) or the end user to specify, otherwise we are likely to be wrong a lot and make things worse

**Comment by mubaraqwahab** [comment]
> We don't know if the link rendered is for the current page or not

I was thinking you could use the active prop to determine this.

> or even if ultimately renders an anchor at all... otherwise we are likely to be wrong a lot and make things worse

True true, I didn't think of this before! Should I go ahead and close this issue?

**Comment by mrazzari** [comment]
Even though a router component might do it, the existing active prop, which renders an "active" class name in the `<a>`, would map 1-1 to `aria-current`. Like you're doing with tabs and aria-selected.
